tests_metricflow/snapshots/test_data_warehouse_tasks.py/str/BigQuery/test_build_metric_tasks__query0.sql

```diff
@@ -1,12 +1,7 @@
--- Aggregate Measures
```

Comment:
Are the comment removal changes intentional?

Reply:
Yea, realize this was unnecessary for this use case, so I changed to return the sql without descriptions